packages/cli/src/test-helpers.ts

```diff
@@ -4,7 +4,7 @@
 //
 // Who can say? ¯\_(ツ)_/¯
 
-import { Package } from "@manypkg/get-packages";
+import type { Package } from "@manypkg/get-packages";
```

Comment:
We should use `erasableSyntaxOnly` + `verbatimModuleSyntax` to catch issues like this

Reply:
I see you have already enabled both 👍